webapp/components/CategoriesSelect/CategoriesSelect.vue

```diff
@@ -9,6 +9,11 @@
       :disabled="isDisabled(category.id)"
       :icon="category.icon"
       size="small"
+      v-tooltip="{
+        content: $t(`contribution.category.description.${category.slug}`),
+        placement: 'bottom-start',
+        delay: { show: 500 },
```

Comment:
The half a second delay feels almost to short for me, because the tooltip is really dominant if it appears … 🤔

Reply:
done